### PR TITLE
Show decoded file URIs

### PR DESCRIPTION
**Pull request by mukul-ms**
Fix for https://github.com/microsoft/sarif-vscode-extension/issues/293 

1) For `Log` field inside `Info` Panel
- Before: 
  - image

- After 
  - image

2) For `Log` tab
- Before: 
   - image

- After
   - image

### CODE REVIEW

**Comment by jeffersonking** [comment]
There is still a URI scheme after the fix. You probably want to take a look at my comment on the issue.

**Comment by mukul-ms** [comment]
> There is still a URI scheme after the fix. You probably want to take a look at my comment on the issue.

Thanks, addressed the feedback.

File: src/extension/loadLogs.ts [hunk 1]
```diff
@@ -21,7 +21,7 @@ export async function loadLogs(uris: Uri[], token?: { isCancellationRequested: b
                 const file = fs.readFileSync(uri.fsPath, 'utf8')  // Assume scheme file.
                     .replace(/^\uFEFF/, ''); // Trim BOM.
                 const {data: log, pointers} = jsonMap.parse(file) as { data: Log, pointers: JsonMap};
-                log._uri = uri.toString();
+                log._uri = uri.scheme === 'file' ? uri.fsPath : uri.toString();
```
Comment (jeffersonking) [inline src/extension/loadLogs.ts:24]: There is a less invasive way to fix this issue than to change the underlying schema.

Comment (mukul-ms) [reply src/extension/loadLogs.ts]: Pushed a fix. Please let me know if it works

File: src/panel/details.tsx [hunk 2]
```diff
@@ -63,8 +64,10 @@ interface DetailsProps { result: Result, height: IObservableValue<number> }
 				<div className="ellipsis">{text ?? '—'}</div>
 				<div className="svSecondary">{uri?.file ?? '—'}</div>
 				<div className="svLineNum">{region?.startLine}:1</div>
-			</>;
-		};
```
Comment (jeffersonking) [inline src/panel/details.tsx:67]: Outside of this PR scope, but what's going on with the whitespace here?

Comment (mukul-ms) [reply src/panel/details.tsx:67]: Weird. pushed a fix in this PR

**Comment by mukul-ms** [comment]
> Recommend refactoring the duplicate code here at some point.

Pushed a fix in this PR

**Pull request merged**